### DISCUSSION TITLE
Megacli on remote server

### DISCUSSION THREAD

**Discussion by plegrand1** [netdata/netdata#17952] [Q&A]
Hello,
I saw that it was possible to use the "Megacli" utility with NetData. 
I wanted to know if it was possible to query a remote machine on which Megacli was installed in order to retrieve data from it.
Thanks for your help

**Comment by ilyam8** [comment]
Hi, @plegrand1. No, it is not possible.

**Reply by plegrand1** [reply]
Ok,
thanks for your answer
